7forecast.tex

```diff
@@ -56,44 +55,43 @@ \subsection{Forecast File Options (forecast.ss)}
   \multicolumn{1}{r}{-9999 0 0 0} & & \\
     & & \\
-    & & \\
-    & & \\
-    & & \\
+    & & \Tstrut\Bstrut\\
+    & & \Tstrut\Bstrut\\
+    % & & \\
 
   \hline
-  0.45 & SPR\textsubscript{target} & \multirow{1}{1cm}[-0.15cm]{\parbox{11cm}{SS3 searches for F multiplier that will produce this level of spawning biomass per recruit (reproductive output) relative to unfished value.}} \Tstrut\\
-    & & \\
+  0.45 & SPR\textsubscript{target} & \multirow{1}{1cm}[-0.15cm]{\parbox{11.3cm}{SS3 searches for the F multiplier that will produce this level of spawning biomass per recruit (reproductive output) relative to the unfished value.}} \Tstrut\Bstrut\\
+    & & \Tstrut\Bstrut\\
   
   \hline
-  0.40 & Relative Biomass Target & \multirow{1}{1cm}[-0.15cm]{\parbox{11cm}{SS3 searches for F multiplier that will produce this level of spawning biomass relative to unfished value. This is not ``per recruit'' and takes into account the spawner-recruitment relationship.}} \Tstrut\\
-    & & \\
-    & & \\
+  0.40 & Relative Biomass Target & \multirow{1}{1cm}[-0.15cm]{\parbox{11.3cm}{SS3 searches for the F multiplier that will produce this level of spawning biomass relative to unfished value. This is not ``per recruit'' and takes into account the spawner-recruitment relationship.}} \Tstrut\Bstrut\\
+    & & \Tstrut\Bstrut\\
     
   \hline 
-  \multicolumn{2}{l}{COND: Do Benchmark = 3} & \multirow{1}{1cm}[-0.15cm]{\parbox{11cm}{B\textsubscript{LIMIT} as a fraction of the B\textsubscript{MSY} where a negative value will be applied as a fraction of B0}} \Tstrut\\
-    & -0.25 & \Bstrut\\
+  \multicolumn{2}{l}{COND: \hyperlink{Bmarks_RefPoints}{Benchmarks} = 3} & \multirow{1}{1cm}[-0.15cm]{\parbox{11.3cm}{B\textsubscript{LIMIT} as a fraction of the B\textsubscript{MSY} where a negative value will be applied as a fraction of B0}} \Tstrut\\
+    & -0.25 & \\
     
     
   \hline
-  0 0 0 0 0 0 0 0 0 0 & Benchmark Years: & \multirow{1}{1cm}[-0.15cm]{\parbox{11cm}{Requires 10 values, beginning and ending years for (1,2) biology (e.g., growth, natural mortality, maturity, fecundity), (3,4) selectivity, (5,6) relative Fs, (7,8) movement and recruitment distribution; (9,10) stock-recruitment parameters for averaging years in calculating benchmark quantities. If there is no time-varying biology it is recommend to select the first model year for the beginning year for biology.}} \Tstrut\\
+  \multirow{1}{1cm}[-0.15cm]{\parbox{2.4cm}{0 0 0 0 0 0 0 0 0 0}} & Benchmark Years: & \multirow{1}{1cm}[-0.15cm]{\parbox{11.3cm}{Requires 10 values, beginning and ending years for (1,2) biology (e.g., growth, natural mortality, maturity, fecundity), (3,4) selectivity, (5,6) relative Fs, (7,8) movement and recruitment distribution; (9,10) stock-recruitment parameters for averaging years in calculating benchmark quantities. The first model year for the beginning year for biology should be selected if there is no time-varying biology.}} \Tstrut\\
```

Comment:
"The first model year for the beginning year for biology ..." is not necessarily easier for me to understand than the previous text.

Reply:
@Rick-Methot-NOAA do you have any suggestion to make this text clearer?

Reply:
Requires 5 pairs of year values over which the mean of derived vectors will be calculated to use in the benchmark (e.g. MSY) calculations.  First pair of years is for biology (e.g., growth, natural mortality, maturity, fecundity); second is selectivity; third is relative Fs among fleets; fourth is movement and recruitment distribution; fifth is stock-recruitment (as the parameters, not as derived quantities). If a factor is not time-varying, select the first model year for the beginning year for the factor else the variance will be artificially reduced.